### ISSUE TITLE
Gibberish on command line on Windows

### ISSUE THREAD

**Issue by TrakeMuldoon** [eginwong/capitalism-ahoy#65]
I paid $18 for this game, and I was expecting much higher quality. 
Every line has little "question mark" boxes. So I can't tell what's going on.

Also, it's easily exploitable, if I quit on the first turn, I always win.

I tried it on Command Line and Powershell.
I tried it on Windows 10, Windows 8, Windows Vista,  and Windows 3.1
But not windows 95, because that's trash.

**Comment by eginwong** [comment]
Good point. Looks like a lot of the emojis aren't displaying correctly on Windows. Thanks for pointing out the bug!

Extra note: GameState.js breaks on certain environments.

```js
    if (!this.turnValues && !this.turnValues.subTurn && !this.turnValues.subTurn.playerId) {
```

It didn't like the `?` syntax apparently.
